crates/python/Cargo.toml

```diff
@@ -18,7 +18,7 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 async-trait = "0.1.73"
-qcs = { path = "../lib", features = ["tracing"] }
+qcs = { path = "../lib", features = ["tracing", "grpc-web"] }
```

Comment:
Right now, this makes the `grpc-web` client the only client the Python library can use. Ideally it should be opt-in, but that would come at the sacrifice of the ergonomics for the Rust crate.